mapper.html

```diff
@@ -0,0 +1,204 @@
+<!--
+Copyright 2022 Google LLC.
+Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="card">
```

Comment:
Let’s make sure the page renders in [no-quirks mode](https://dom.spec.whatwg.org/#concept-document-no-quirks): `<!DOCTYPE html>`

Reply:
done